cpp/roaring.hh

```diff
@@ -91,6 +95,16 @@ class Roaring {
         addMany(l.size(), l.begin());
     }
 
+    /**
+     * Construct a roaring object by taking control of a malloc()'d C struct.
```

Comment:
This is just a position moving?

Reply:
Yes, to keep the same order as the 32-bit version.